src/test/java/ca/corefacility/bioinformatics/irida/ria/integration/pages/projects/NcbiExportPage.java

```diff
@@ -52,9 +52,9 @@ public NcbiExportPage(WebDriver driver) {
 	public static NcbiExportPage goTo(WebDriver driver, Long projectId, Collection<Long> sampleIds) {
 
 		StringJoiner stringJoiner = new StringJoiner("&");
-		sampleIds.forEach(s -> stringJoiner.add("ids[]=" + s.toString()));
+		sampleIds.forEach(s -> stringJoiner.add("ids%5B%5D=" + s.toString()));
```

Comment:
Do you maybe want to use a url encoder to make more robust.  Possibly create a utility function? https://www.baeldung.com/java-url-encoding-decoding

Reply:
Not sure that it is entirely necessary, just because this was the only instance in the integration tests where `[]` were used in a URL. This change wasn't specifically required for this branch, it was necessary for my `spring-boot` branch.

I could change `AbstractPage#get` to perform the following on the url though:
```
String url = null;
try {
	url = URLEncoder.encode(BASE_URL + relativeUrl, "UTF-8");
} catch (UnsupportedEncodingException e) {
	throw new RuntimeException(e.toString());
}
```

Reply:
Actually that does not work as intended. It produces a bad url.

Reply:
Addressed in 8fb6261